MobileWallet/Screens/Send/AddNote/AddNoteViewController.swift

```diff
@@ -74,16 +74,18 @@ class AddNoteViewController: UIViewController, UITextViewDelegate, SlideViewDele
         super.viewWillAppear(animated)
         styleNavigatorBar(isHidden: false)
 
-        guard let pubKey = publicKey else { return }
-
-        do {
-            try showNavbarEmojies(pubKey)
-        } catch {
-            UserFeedback.shared.error(
-                title: NSLocalizedString("Public key error", comment: "Add amount view"),
-                description: NSLocalizedString("Failed to get Emoji ID from user's contact", comment: "Add amount view"),
-                error: error
-            )
+        if title == nil {
```

Comment:
I think this should be something similar to `if contact == nil`. Or maybe an `Either` data structure like [this](https://www.mokacoding.com/blog/swift-either/).